sam/app/template.yaml

```diff
@@ -57,6 +64,13 @@ Parameters:
     Description: GitHub repo branch name. It defaults to master if not specified.
     Type: String
     Default: master
+  CodeCommitRepoName:
```

Comment:
Call this `CodeCommitRepo` to be consistent with `GitHubRepo`?

Reply:
Fair enough